spec/controllers/documents_controller_spec.rb

```diff
@@ -165,7 +165,7 @@
       }.to change{Document.count}.by -1
 
       expect(response).to redirect_to "/projects/#{project.identifier}/documents"
-      expect{Document.find(document.id)}.to raise_error
+      expect{Document.find(document.id)}.to raise_error ActiveRecord::RecordNotFound
```

Comment:
Line is too long. [84/80]<br>Space missing to the left of {.<br>Space missing inside {.<br>Space missing inside }.